peps/pep-0710.rst

```diff
@@ -437,6 +437,18 @@ contain any entries. In such cases, pip does not create any
 is encouraged for consumers to rebuild wheels with a newer version of pip in
 these cases.
 
+uv developers `raised a concern about requiring at least one hash
+<https://discuss.python.org/t/25428/34>`__ in the ``provenance_url.json`` file
+as uv does not calculate distribution hashes unless explicitly required.
+However, requiring at least one hash aids in integrity checks for
+distributions. This is important in scenarios involving lock files or when
+identifying distributions as part of SBOMs. The ``provenance_url.json`` file
+mandates the inclusion of at least one hash for the downloaded distribution.
+Installers that do not compute hashes of distributions as part of the
+installation process (e.g., due to performance reasons) can omit creating the
+``provenance_url.json`` file. However, the limitations affecting the
+auditability of Python environments should be taken into account.
```

Comment:
I think either remove this sentence or expand upon it (what limitations?). If you do keep it, note that the sentence on Line 443 also starts with "However,", so this one could be rephrased.